tests/kafkatest/tests/streams/base_streams_test.py

```diff
@@ -13,22 +13,40 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from ducktape.tests.test import Test
 from ducktape.utils.util import wait_until
 from kafkatest.services.verifiable_consumer import VerifiableConsumer
 from kafkatest.services.verifiable_producer import VerifiableProducer
-from kafkatest.tests.kafka_test import KafkaTest
+from kafkatest.services.zookeeper import ZookeeperService
+from kafkatest.services.kafka import KafkaService, quorum
 
 
-class BaseStreamsTest(KafkaTest):
+class BaseStreamsTest(Test):
     """
     Helper class that contains methods for producing and consuming
     messages and verification of results from log files
 
     Extends KafkaTest which manages setting up Kafka Cluster and Zookeeper
     see tests/kafkatest/tests/kafka_test.py for more info
     """
-    def __init__(self, test_context,  topics, num_controllers=1, num_brokers=3):
-        super(BaseStreamsTest, self).__init__(test_context, num_controllers, num_brokers, topics)
+    def __init__(self, test_context, topics, num_controllers=1, num_brokers=3):
+        self.num_zk = num_controllers
+        self.num_brokers = num_brokers
+        self.topics = topics
+
+        self.zk = ZookeeperService(test_context, self.num_zk) if quorum.for_test(test_context) == quorum.zk else None
```

Comment:
Why this? I thought ZK is gone?

Reply:
I'm just preserving the code that is already there (in KafkaTest). the quorum parameter is still everywhere in the streams tests. I can push a clean-up PR, for this PR, I just preserved what's tehre.

Reply:
Okay, removed it. I suppose quorum=zk is not used, so no need to support it here.

Reply:
Thanks. Might be a good idea to also go over other tests and clean this up, if it's still there?

I would only expect some upgrade/downgrade tests which use older broker versions to still use ZK parameter.